sdk/mx.sdk/mx_sdk_vm_impl.py

```diff
@@ -1231,8 +1250,10 @@ def contents(self):
             ]
             if _debug_images():
                 build_args += ['-ea', '-O0', '-H:+PreserveFramePointer', '-H:-DeleteLocalSymbols']
-            if _get_svm_support().is_debug_supported(image_config):
+            if _get_svm_support().generate_debug_info(image_config):
```

Comment:
* `-g` is now passed by default also while building GraalVM CE's native images on linux and windows (like what's already happening for EE)
* It's also passed on darwin if the user explicitly requests `--debug-images`